Improve readme (#3)
ambanum/media-scale: README.md

# Media-scale

> Compare one article's visibility on social media to known event's visibility in its regional's reference media to put number of reactions into perspective.

Reviewer: ```suggestion
Compare an article's visibility on social media to other known events visibility in its local reference media to give some perspective to the number of reactions.
```

> ## Motivation, buts et usages

> Dans le cadre de la lutte contre la désinformation, les analyses et vérifications menées reprennent régulièrement - voire se basent sur - des **données chiffrées pour mesurer l'impact d'un contenu** sur une société. Il peut s'agir par exemple du nombre de réactions à ce contenu, du nombre de commentaires, ou encore du nombre de fois qu'un contenu a été partagé.

> Or, ces données sont rarement **mises en perspective**. Plutôt que de se focaliser sur leur aspect quantitatif, nous proposons de les organiser en fonction d'une échelle de pertinence. Ainsi, en considérant le sujet du contenu plutôt que son nombre de partages, on aboutit à une **échelle de référence qualitative**.

> Sous la forme d'un tableur (fichier au format .csv), le nombre d'engagements (partages, likes et commentaires) suscités par un article est lié à :
> - son pays d'origine,
> - sa source (principaux journaux nationaux),
> - sa date de publication,
> - son sujet (titre).

> Il y a donc une **contextualisation et une mise en perspective de données chiffrées**, facilement **lisibles et accessibles**.

> The comparison can be done manually or with the exposed API.

Reviewer: This English sentence on its own feels pretty weird. Either include it above the French intro, or after the API header?

## API

> Available at [https://disinfo.quaidorsay.fr/api/media-scale/1.0](https://disinfo.quaidorsay.fr/api/media-scale/1.0)

Reviewer: ```suggestion
A public test instance is available at [https://disinfo.quaidorsay.fr/api/media-scale/1.0](https://disinfo.quaidorsay.fr/api/media-scale/1.0)
```

}
```

- - -

## Development

> [Node](https://nodejs.org/en/) is required as this API is built on top of it.

Reviewer: ```suggestion
This API is built with [Node](https://nodejs.org/en/). You will need to [install it](https://nodejs.org/en/download/) to run this API.
```

That's it.

## Deployment

Clone the repository on your server, install dependencies and run the webserver.

> We suggest to use a production process manager for Node.js like [pm2](https://github.com/Unitech/pm2) or [Forever](https://github.com/foreversd/forever#readme)

Reviewer: ```suggestion
We suggest to use a production process manager for Node.js like [pm2](https://github.com/Unitech/pm2) or [Forever](https://github.com/foreversd/forever#readme).
```
